app/activity/[id]/components/activity-card.tsx

```diff
@@ -0,0 +1,82 @@
+'use client'
+
+import { Heading, Box, Flex, Spacer, Text, IconButton } from '@chakra-ui/react'
+import { FiClock, FiMapPin, FiLink2, FiEdit3, FiTrash2 } from 'react-icons/fi'
+import { Link } from '@/components/link'
+import { customColors } from '@/theme/color'
+import { ActivityCollectionQuery } from '@generated/api'
+
+type ActivityInfoProps = {
```

Comment:
I think the name is confusing? In this case, ActivityCardProps is correct. But I think this components is actually not card so it's better to change to `Activity Header` or something

Reply:
I changed the name!